doc/changes.md

```diff
@@ -1,5 +1,128 @@
 # Changelog
 
+## Release v0.11.0 (20YY/MM/DD)
+
+[![Digital Object Identifier for PyGMT v0.11.0](https://zenodo.org/badge/DOI/10.5281/zenodo.3781524.svg)](https://doi.org/10.5281/zenodo.3781524)
+
+### Highlights
+
+*
```

Comment:
Here are some ideas for the Highlights section:
```suggestion
* Intro for table inputs in PyGMT
* Gallery example for Chorophleth map
* Easy access to remote datasets of planetary reliefs (Moon, Venus, Mars, Mercury, Pluto)
* `Figure.savefig`: Support generating GeoTIFF files
* `Figure.text`: Allow lists as input type for the `angle`, `font`, and `justify` parameters
* Start adding type hints support
```
Feel free to select the preferred items (I feel 2-4 highlights are a good number) and to improve the formulations.

Reply:
Maybe omit the fig.savefig one, and leave out the type hints too (there's not many of them yet).